### ISSUE TITLE
wikiless

### ISSUE THREAD

**Issue by ghost** [nsh07/WikiReader#40]
support wikiless https://github.com/Metastem/wikiless/wiki/Instances
Some countries have blocked Wikipedia, we can use the Wikiless frontend to access

**Comment by nsh07** [comment]
This seems a bit out of scope to me, I will do a bit of research on this and reply here

**Comment by GfEW** [comment]
> This seems a bit out of scope to me, I will do a bit of research on this and reply here

Any news to this?

Given that
* there are valid [reasons to prefer wikiless](https://github.com/Metastem/Wikiless/wiki/FAQ) over wikipedia
* wikiless doesn't support editing, so there is one valid reason left to resort to original wikipedia, to contribute to wikipedia's content (should you want to)
* WikiReader doesn't support editing, anyway, so there is also one valid reason left to resort to original wikipedia for edits (e. g. wikipedia.org in a web browser), which coincides with the above,

it would make a lot of sense if you could configure WikiReader to use wikiless.

**Comment by nsh07** [comment]
> 4. the practical limitations given by 2. and 3. coincide, so WikiReader would strictly enhance wikiless, and vice versa,
> 
> it would make a lot of sense if WikiReader could be configured to use wikiless.

I still don't get the point of this project. The entire motivation behind this project seems to be the fact that the author is terribly paranoid about the NSA tracking people down, which is a valid concern but it is too far a step in the name of "privacy" in my opinion. Have a look at [this comment](https://www.reddit.com/r/privacytoolsIO/comments/p3ez7y/comment/h8qtpqy/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).

Also, this was the practicality side of things, now coming to the technical side, the server is terribly slow and the API responds with an HTML document containing the JSON inside the <body> which is really annoying to parse into the actual extracted JSON. Also APIs using the /api/rest_v1 endpoint seem to be completely unavailable so the feed cannot work with this.

And to top this all off, the URL format is different (instead of {language}.wikipedia.org it is wikiless.com/{whatever url}?lang={language}) which will completely new network code to load

TLDR: I am still not interested in implementing this because I don't see the point. To protect your IP address or use Wikipedia from a blocked country, you can just use a free VPN or Tor, so the point of this is moot

**Comment by SomeRandomCpu** [comment]
This is completely unnecessary. Wikipedia has not been hijacked and I think the author of wikiless has severe paranoia.

If you're that worried, just use a VPN
